server/routes/userRoutes.js

```diff
@@ -1,9 +1,14 @@
 import express from "express";
-import { authUser, registerUser } from "../controllers/userController.js";
+import {
+  authUser,
+  deleteUser,
+  registerUser,
+} from "../controllers/userController.js";
 
 const router = express.Router();
 
 router.post("/", registerUser);
 router.post("/auth", authUser);
+router.delete("/:id", deleteUser);
```

Comment:
Let's change this to router.route("/:id").delete(deleteUser)

Reply:
Updated!